modules/cnv-storage-wizard-fields-web.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_users_guide/TBD
+
+[id="cnv-storage-wizard-fields-web_{context}"]
+== Storage fields
+|===
+|Name | Description
+
+|Create Disk
+|Create a new disk for the virtual machine.
+
+|Attach Disk
+|Select an existing disk, from a list of available PVCs, to attach to the virtual machine.
+
+|DISK NAME
```

Comment:
Why is the format different for some of these parameters?

Are these UI elements? The should be in bold if they are or sentence case if they're not.